### PR TITLE
[kotlin cleanUp]  AudioRecorder.kt,AudioView.kt,StatsMetaInfo.kt

### LINKED ISSUE #10489: Kotlin Cleanup
Currently, there are 84 `@KotlinCleanup` in AnkiDroid code source. Most of those changes are supposed to be simple to do and would be a good pull request by themselves. Also, if you are a java developer and new to Kotlin, it will be a great way to learn Kotlin advantages, since it's usually tasks that were introduced when translating java to Kotlin, and which can actually be improved in Kotlin compared to original java code.

Ideally, each PR should contains a single cleanup. This ensure the PR is very very quick to review and merge. The main exception would be if a clean-up is dependent on another clean-up and this would cause merge conflict.

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
**kotlincleanUp of AudioRecorder.kt,AudioView.kt,StatsMetaInfo.kt**

## Fixes
Fixes some of #10489 

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt [hunk 1]
```diff
@@ -18,157 +18,155 @@ package com.ichi2.anki.stats
...
+    private var mcount = 0.0
+    private var seriesList: Array<DoubleArray>? = null
     var isStatsCalculated = false
     var isDataAvailable = false
     fun getmSeriesList(): Array<DoubleArray>? {
-        return mSeriesList
+        return seriesList
     }
 
     fun setmSeriesList(seriesList: Array<DoubleArray>) {
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt:49]: All of these can be changed: make the `var` public and remove the methods (and `"JvmName` if necessary)

**Review by david-allison — CHANGES_REQUESTED** [review]
Thanks! Little work on `StatsMetaInfo`, rest are great!

File: AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt (hunk 1 above)
Comment (Prince-kushwaha) [reply AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt:49]: @david-allison I think properties are private is good becouse property are modified by  setter and acesss by getter method   setter and getter method make easy to access and modify value

Comment (Prince-kushwaha) [reply AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt:49]: i think their is not need to remove setter and getter method

File: AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt [hunk 2]
```diff
@@ -18,157 +18,155 @@ package com.ichi2.anki.stats
 
 import android.annotation.SuppressLint
 import com.ichi2.libanki.stats.Stats.AxisType
-import com.ichi2.utils.KotlinCleanup
 
 /**
  * Interface between Stats.java and AdvancedStatistics.java
  */
-@KotlinCleanup("Remove field prefixes, make properties private")
 @SuppressLint("VariableNamingDetector")
```
[the remark this answers is not in the document]
Comment (Prince-kushwaha) [reply AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt]: i did  kotlinCleanUp to remove suppression

File: AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt (hunk 1 above)
Comment (david-allison) [reply AnkiDroid/src/main/java/com/ichi2/anki/stats/StatsMetaInfo.kt:47]: But the setter methods are equivalent to the property accessors, just with a lot more code, and worse names?

**Review by david-allison — APPROVED** [review]
We can rebase this if commits 1 and 2 are squashed, otherwise squash merge

**Review by mikehardy — APPROVED** [review]
Nice! Thank you @Prince-kushwaha - the java formatting in AdvancedStatistics is not quite to our normal coding conventions but as we approach the end of our Java to Kotlin migration I'm inclined to let it go. Time is better spent actually migrating the code instead...

**Pull request merged**